inherited_resources.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.3'
 
   s.add_dependency("responders")
-  s.add_dependency("actionpack", ">= 5.0", "< 5.3")
-  s.add_dependency("railties", ">= 5.0", "< 5.3")
+  s.add_dependency("actionpack", ">= 5.0", "< 6.0")
+  s.add_dependency("railties", ">= 5.0", "< 6.0")
```

Comment:
I was thinking about this a bit more, and I think this is a good strategy.

We allow any prereleases for now, but not the final version. We'll watch for compatibility from now on but hold off a new release until Rails 6.0 is released and we confirm we are compatible. Then bump the upper limit here to 6.1 for example and release a new version